src/styles/ReadState.scss

```diff
@@ -5,5 +5,5 @@
   position: absolute;
```

Comment:
doesn't look like the styles in this file are applied anywhere so we should delete